api/src/services/setup/upgrade-steps.js

```diff
@@ -116,7 +116,20 @@ const complete = async (buildInfo) => {
   const stagingDoc = await upgradeUtils.getStagingDoc(buildInfo);
   const payload = upgradeUtils.getUpgradeServicePayload(stagingDoc);
 
-  return await upgradeUtils.makeUpgradeRequest(payload);
+  const response = await upgradeUtils.makeUpgradeRequest(payload);
+
+  if (!upgradeUtils.upgradeResponseSuccess(payload, response)) {
+    logger.error('No compose files were updated: %o', response);
+    throw new Error('No compose files were updated');
```

Comment:
I presume this shows the generic error message in the admin UI? If so, we should improve this error message to have more information about what went wrong and how to fix it.